### ISSUE TITLE
Language Switcher default Link Text value

### ISSUE THREAD

**Issue by Dinamiko** [inpsyde/MultilingualPress#262]
#### Version Information

* PHP: any
* WordPress: Latest 
* MLP: 2.6.x

#### Steps to Reproduce

1. WordPress Multisite installation with at least 2 connected sites.
2. At least one Post translated to both languages.
3. Go to Appearance / Widgets and drag and drop Language Switcher on a Widget area.
4. Click Visit Site on Admin Bar.

#### What I Expected

Language Switcher should be shown in the sidebar. 

#### What Happened Instead

Language Switcher doesn't shown in the sidebar.

#### Why?

![01](https://user-images.githubusercontent.com/456223/27391074-3965e566-56a3-11e7-9344-7ad282d1c8f9.jpg)

Because default Link Text value is None.

**Comment by Dinamiko** [comment]
I forgot to add #issue-number in the commit, sorry.
https://github.com/inpsyde/MultilingualPress/commit/535af0e08e6d9ebaa912dcd991a4560170b7ccee

I tried to cover these 2 cases:

**Case 1**
As an admin I want to see the Language Switcher in the frontend when I add it to a widget area.
 
**Case 2**
As an admin with an already in use Language Switcher configured as:
Link text: None and Show flag: Checked
I want to continue seeing the same after Case 1 is fixed.

![03](https://user-images.githubusercontent.com/456223/27427333-e45347ac-573e-11e7-805a-8439265d985f.jpg)
